build.gradle.kts

```diff
@@ -20,11 +20,16 @@ plugins {
   application
 }
 
-java.sourceCompatibility = JavaVersion.VERSION_17
+val ecommerceCommonsVersion = "0.19.5"
+
+val ecommerceCommonsRef = ecommerceCommonsVersion
```

Comment:
Is this double value necessary?

Reply:
I reported this duplication also in README, it has been added on purpose in order to set it to a feature branch during development phases (the same we already do with scm maven plugin). 
In normal usage refs is the same as version

Reply:
Refactored ecommerceCommonsRef variable name with https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/5/commits/ec447da1f728f07a0c9aeccef75f657ae892da28